pyproject.toml

```diff
@@ -0,0 +1,30 @@
+[tool.poetry]
+name = "taskiq-redis"
+version = "0.1.0"
+description = ""
+authors = ["taskiq-team"]
+
```

Comment:
If you want to follow [PEP-420](https://peps.python.org/pep-0420/) you have to add this lines here:

```
packages = [{ include = "taskiq", from = "taskiq_redis" }]
```

Also your directory name is incorrect. Please rename it to have underscores instead of hyphens.
So this `taskiq-redis` will become this `taskiq_redis`, because python cannot import packages with hyphens it throws syntax errors.

 But please do not rename package in pyproject.toml. Because package name is good.

Reply:
Added, thanks)